src/ray/raylet/lineage_cache.cc

```diff
@@ -35,20 +37,28 @@ const TaskID LineageEntry::GetEntryId() const {
   return task_.GetTaskSpecification().TaskId();
 }
 
-const std::unordered_set<TaskID> LineageEntry::GetParentTaskIds() const {
+const std::unordered_set<TaskID> &LineageEntry::GetParentTaskIds() const {
+  return parent_task_ids_;
+}
+
+void LineageEntry::ComputeParentTaskIds() {
+  parent_task_ids_.clear();
   std::unordered_set<TaskID> parent_ids;
```

Comment:
This can be removed I think

Reply:
Ah, thanks! Not sure how that compiled...